test/sim/pokedex.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+const assert = require('./../assert');
+
+describe('Pokedex', function () {
+	it('should make sure every Pokemon name is correct', function () {
+		for (const pokemonid in Dex.data.Pokedex) {
+			const name = Dex.data.Pokedex[pokemonid].name;
+			const id = toID(name);
+			assert(pokemonid === id, `Mismatched Pokemon key "${pokemonid}" of ${name}`);
```

Comment:
```suggestion
		const Pokedex = Dex.data.Pokedex;
		for (const pokemonid inPokedex) {
			const entry = Pokedex[pokemonid];
			assert.equal(toID(entry.name), pokemonid, `Mismatched Pokemon key "${pokemonid}" of ${entry.name}`);
			if (entry.evos) {
				for (const evo of entry.evos) {
					assert.equal(evo, (Pokedex[toID(evo)] || {}).name, `Mismatched evo "${evo}" of ${entry.name}`);
				}
			}
```

(If we bump min version to Node 14, we can use `?.` in tests!)

Reply:
Anyway yeah, do this, but also for prevo, cosmeticFormes, otherFormes, and changesFrom.